docs/dev-guide/contributing/bugs.rst

```diff
@@ -138,6 +140,46 @@ A target release can also be set during triage. An issue that has target release
 will block a release. Once triaged, the bug is included in the `Prioritized Bugs query`_ and awaits
 a developer to pick it up.
 
+Triage Issue Fields
+^^^^^^^^^^^^^^^^^^^
+
+The Priority field represents the order in which issues will be taken from the list of prioritized
+bugs, with higher priority issues generally being taken before lower priority issues.
+
+Priorities are defined as follows.
+
+========    ===============================================================================
+Priority    Description
+========    ===============================================================================
+Urgent      Extremely important, such as issues that must be fixed before the next release.
+High        Very important, but not blocking the next release.
+Normal      Average importance, generally worked on after High Priority issues.
```

Comment:
I am not sure if it worth to mention whether issue is  blocking release or not. Sometimes issue with high priority can be considered as blocker also. I think we need to re-phrase this somehow